src/MBrace.Streams/CloudStreams.fs

```diff
@@ -703,5 +703,54 @@ module CloudStream =
         sources
         |> ofCloudFiles CloudFileReader.ReadLines
         |> flatMap Stream.ofSeq
-        
+
+    /// <summary>
+    /// Constructs a CloudStream of lines from a path.
+    /// </summary>
+    /// <param name="path">The path to the text file.</param>
+    let ofTextFile (path : string) : CloudStream<string> =
```

Comment:
It's certainly good to be using Spark as a comparison point.  The Spark model seems fairly biased to line-by-line processing - which is OK of course since Hadoop is too and processing log files is of huge importance for these systems - but I suspect the MBrace programming model may find a broader (or just different) range of sweet-spot applications over time, and so being a bit more explicit about Line-by-Line processing is ok.

BTW I think you should also rename ofCloudFilesByLines --> ofCloudFilesByLine too (depluralize) 

cheers!